sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1980,32 +2257,42 @@ public void testRemoveTemporaryTables() throws Exception {
   }
 
   @Test
-  public void testRuntimeOptionsNotCalledInApplyInputTable() {
+  public void testRuntimeOptionsNotCalledInApplyInputTable() throws IOException {
```

Comment:
This test was just plain broken before (it tried to access a gs:// location which failed) - I'm not sure why it wasn't always failing. BigQueryIO doesn't try and access the gs location if a job service is set.